src/mlem.rs

```diff
@@ -216,6 +220,25 @@ impl Image {
     }
 }
 
+/// Define the scatter prediction histogram (incomplete)
+pub fn scatter_prediction<T>(lors: &[LOR]) -> Option<Scattergram<T>>
+where
+    T: Lorogram
```

Comment:
This is wrong but I've still not worked out how to make it accept anything that implements `Lorogram`